pytorch/training/docker/1.7.0/py3/example/Dockerfile.gpu

```diff
@@ -0,0 +1,12 @@
+
+# Use the Deep Learning Container as a base Image
+ARG BASE_IMAGE=""
+
+FROM $BASE_IMAGE
+
+RUN pip install tensorboardX==1.6.0
```

Comment:
Are there any updates needed to tensorboardX version?

Reply:
Changed to version 2.1